tests/kwargs/test_generator_dependencies.py

```diff
@@ -170,9 +169,9 @@ async def nested_dependency_two(generator_dep: str, nested_one: str) -> str:
     @get(
         "/",
         dependencies={
-            "generator_dep": Provide(dependency, sync_to_thread=False),
-            "nested_one": Provide(nested_dependency_one, sync_to_thread=False),
-            "nested_two": Provide(nested_dependency_two, sync_to_thread=False),
+            "generator_dep": dependency,
+            "nested_one": nested_dependency_one,
+            "nested_two": nested_dependency_two,
```

Comment:
These ones aren't generators, right?

Reply:
They're not, but they're async so the `sync_to_thread=False` was superfluous